File: src/main/java/com/back/nbe9112team06/domain/member/dto/response/AvailabilityResponse.java
package com.back.nbe9112team06.domain.member.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;

// 이메일 중복여부 체크 dto
@Schema(description = "이메일 사용 가능 여부 응답 데이터")
public record AvailabilityResponse(
        @Schema(description = "이메일 사용 가능 여부 (true: 사용 가능, false: 중복)", example = "true")
        boolean available
) {}

File: src/main/java/com/back/nbe9112team06/domain/member/dto/response/AvailabilityResponse.kt
package com.back.nbe9112team06.domain.member.dto.response

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "이메일 사용 가능 여부 응답 데이터")
data class AvailabilityResponse(

    @field:Schema(description = "이메일 사용 가능 여부 (true: 사용 가능, false: 중복)", example = "true")
    val available: Boolean,
)